src/main/java/appeng/block/AEBaseBlock.java

```diff
@@ -258,25 +215,21 @@ public IIcon getIcon( int direction, int metadata )
 		return this.getRendererInstance().getTexture( ForgeDirection.getOrientation( direction ) );
 	}
 
+	public ICustomCollision getCustomCollision( World w, int x, int y, int z )
```

Comment:
`protected` should be sufficent. The code currently using it was already dealing with `null`, so NPE will only be a issue with future code. Maybe annotating it `@Nullable` to indicate it.

Otherwise it is really out of scope for this PR and should be dealt separately in one refactoring collisions, if it is really an issue.